test/functional/test-render-delaying-extensions.js

```diff
@@ -43,13 +45,15 @@ describe('waitForExtensions', () => {
 
   afterEach(() => {
     sandbox.restore();
+    window.document.head.innerHTML = '';
   });
 
   it('should return undefined if no extension is presented', () => {
     expect(waitForExtensions(win)).to.equal(undefined);
   });
 
-  it('should keep waiting when some extensions are not ready yet', () => {
+  it('should timeout if some extensions do not load', () => {
+    win = window; // Use the main window to be able to tick the clock
```

Comment:
Haha, that's something I was looking for. Should have asked you folks earlier :-)